build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -55,6 +55,19 @@ spec:
                   minimum: 1
                   maximum: 65535
                   default: 179
+                confederation:
+                  type: object
+                  properties:
+                    enable:
+                      type: boolean
+                      default: false
+                    identifier:
+                      type: integer
+                      format: int32
+                    peers:
+                      type: array
+                      items:
+                        type: integer
```

Comment:
should there be a minimum and maximum value here like for `asn` (and also we are missing `format`)?

Reply:
Looks like the first comment from Antonin about the limitation is not addressed:
Should we add the same minimum and maximum as `localASN` to `1-65535`?

Reply:
Done. Thanks for reminding it.